core/templates/components/forms/schema-based-editors/schema-based-list-editor.directive.ts

```diff
@@ -142,6 +168,7 @@ angular.module('oppia').directive('schemaBasedListEditor', [
                 SchemaDefaultValueService.getDefaultValue($scope.itemSchema()));
               FocusManagerService.setFocus(
                 $scope.getFocusLabel($scope.localValue.length - 1));
+              updateOldToNewListMappingOnAddElement();
```

Comment:
Here, you call it AddElement; below, you call it just Delete. To maintain consistency, should you add Element to the second one, so that it becomes DeleteElement?

Reply:
done